torchtext/datasets/trec.py

```diff
@@ -0,0 +1,109 @@
+import os
+
+from .. import data
+from six.moves import urllib
+
+
+class TREC(data.ZipDataset):
```

Comment:
I don't think you're using any methods from `ZipDataset`? Would this work the same if it just inherited from `Dataset`?